CorsixTH/Lua/game_ui.lua

```diff
@@ -79,9 +80,6 @@ function GameUI:GameUI(app, local_hospital, map_editor)
   self.transparent_walls = false
   self.do_world_hit_test = true
 
-  self:setRandomAnnouncementTarget()
-  self.ticks_since_last_announcement = 0
```

Comment:
Do I need to clean these up, so they're not in the savegames anymore?

Reply:
Preferably yes, in the afterLoad.